### ISSUE TITLE
CoordiNet: uncertainty-aware pose regressor for reliable vehicle localization. (arXiv:2103.10796v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#4906]
CoordiNet: uncertainty-aware pose regressor for reliable vehicle localization. (arXiv:2103.10796v1 [cs.CV])<br>
https://ift.tt/3scUoEv<br>
<p>In this paper, we investigate visual-based camera localization with neural networks for robotics and autonomous vehicles applications. Our solution is a CNN-based algorithm which predicts camera pose (3D translation and 3D rotation) directly from a single image. It also provides an uncertainty estimate of the pose. Pose and uncertainty are learned together with a single loss function. Furthermore, we propose a new fully convolutional architecture, named CoordiNet, designed to embed some of the scene geometry.</p>
<p>Our framework outperforms comparable methods on the largest available benchmark, the Oxford RobotCar dataset, with an average error of 8 meters where previous best was 19 meters. We have also investigated the performance of our method on large scenes for real time (18 fps) vehicle localization. In this setup, structure-based methods require a large database, and we show that our proposal is a reliable alternative, achieving 29cm median error in a 1.9km loop in a busy urban area.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
